test/Microsoft.DotNet.Tools.Builder.Tests/IncrementalTestBase.cs

```diff
@@ -28,9 +28,8 @@ public IncrementalTestBase(string testProjectsRoot, string mainProject, string e
             // create unique directories in the 'temp' folder
             var root = Temp.CreateDirectory();
 
-            // recursively copy projects to the temp dir and restore them
+            // recursively already restored copy projects to the temp dir
```

Comment:
I don't think this comment is needed at all. Just delete it. CopyDirectory already states that. Plus, I am not a big fan of comments in general. We can have that conversation some day. It is always interesting.